css/packages/stylelint-config-nebenan/package.json

```diff
@@ -21,14 +21,14 @@
   "devDependencies": {
     "eslint": "^6.7.2",
     "eslint-config-nebenan-base": "^1.0.2",
-    "stylelint": "^12.0.0"
+    "stylelint": "^13.2.1"
   },
   "dependencies": {
     "stylelint-config-standard": "^19.0.0",
     "stylelint-order": "^3.1.1",
-    "stylelint-scss": "^3.13.0"
+    "stylelint-scss": "^3.14.2"
   },
   "peerDependencies": {
-    "stylelint": "^11.0.0 || ^12.0.0"
+    "stylelint": "^12.0.0 || ^13.0.0"
```

Comment:
I thought that sylelint-scss does not support anything lower than stylelint 12 - I was wrong. Anyhow, I updated stylelint v11 to v13 in packages depending on stylelint-config-nebenan without any major problems so I think it would be okay to keep it in.